server/image_status.go

```diff
@@ -9,8 +9,20 @@ import (
 // ImageStatus returns the status of the image.
 func (s *Server) ImageStatus(ctx context.Context, req *pb.ImageStatusRequest) (*pb.ImageStatusResponse, error) {
 	logrus.Debugf("ImageStatus: %+v", req)
-	// TODO
-	// containers/storage will take care of this by looking inside /var/lib/ocid/images
-	// and getting the image status
-	return &pb.ImageStatusResponse{}, nil
+	image := ""
+	img := req.GetImage()
+	if img != nil {
+		image = img.GetImage()
+	}
+	status, err := s.images.ImageStatus(ctx, image)
```

Comment:
Was the image already pulled when you encountered this error?  If not, if you manually pull it, does that make the error stop occurring?